benchmarks/utils/benchmark.py

```
@@ -9,15 +9,28 @@
-                f"Input '{label}' must be an HTTP endpoint (starting with http:// or https://). Got: {value}"
-            )
+        # Check if running in cluster and allow internal service URLs
+        if is_running_in_cluster():
+            # When running in cluster, allow internal service URLs (e.g., service.namespace.svc.cluster.local:port)
+            if not (
+                value.lower().startswith(("http://", "https://"))
+                or (":" in value and not value.startswith("/"))
+            ):
+                raise ValueError(
+                    f"Input '{label}' must be an HTTP endpoint or internal service URL (e.g., service.namespace.svc.cluster.local:port). Got: {value}"
+                )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)